dotCMS/src/main/java/com/dotmarketing/portlets/categories/business/CategoryFactory.java

```diff
@@ -224,7 +224,7 @@
 	 * Deletes all the top level categories
 	 * @return
 	 */
-	abstract protected void deleteTopLevelCategories();
+	abstract protected void deleteTopLevelCategories() throws DotDataException;

```

Comment:
isn't possible to wrap in a runtime exception?

Reply:
This method is about to be dropped. since it wasn't used at all.